ops/terraform/services/server/server-regression/main.tf

```
@@ -155,3 +157,58 @@ resource "aws_lambda_function" "glue_trigger" {
 
   role = aws_iam_role.glue_trigger.arn
 }
+
+resource "aws_lambda_function" "spice_refresh_trigger" {
+  description      = "Triggers a full SPICE refresh when the server regression glue crawler completes"
+  architectures    = ["x86_64"]
+  function_name    = "bfd-${local.env}-${local.service}-refresh-spice-trigger"
+  handler          = "spice-trigger.lambda_handler"
+  memory_size      = 128
+  package_type     = "Zip"
+  role             = aws_iam_role.spice_refresh_role.arn
+  runtime          = "python3.9"
+  filename         = data.archive_file.spice_trigger.output_path
+  source_code_hash = data.archive_file.spice_trigger.output_base64sha256
+  timeout          = 3
+  environment {
+    variables = {
+      ENV = local.env
+    }
+  }
+}
+
+resource "aws_cloudwatch_event_rule" "spice_refresh_event_rule" {
+  description = "Event to execute a refresh of SPICE server regression ${local.env} dataset"
+  event_pattern = jsonencode(
+    {
+      detail = {
+        crawlerName = [
+          "bfd-insights-bfd-${local.env}-server-regression",
+        ]
+        state = [
+          "Succeeded",
+        ]
+      }
+      detail-type = [
+        "Glue Crawler State Change",
+      ]
+      source = [
+        "aws.glue",
+      ]
+    }
+  )
+  name = "bfd-${local.env}-${local.service}-refresh-spice-event"
+}
+
+resource "aws_cloudwatch_event_target" "spice_refresh_event_target" {
+  arn  = aws_lambda_function.spice_refresh_trigger.arn
+  rule = aws_cloudwatch_event_rule.spice_refresh_event_rule.id
+}
+
+resource "aws_lambda_permission" "allow_event_bridge_access" {
+  statement_id  = "AWSEvents_bfd-${local.env}-server-regression-refresh-spice-event"
```

```suggestion
  statement_id  = "${local.spice_trigger_lambda_name}-on-glue-crawler-success"
```

nit: Only accept this suggestion if the previous suggestion regarding the name of the Event Rule is acceptable. This is not exactly important, and is more stylistic than anything, but it does make the relationship more obvious between the Lambda, the Event Rule, and this invocation resource.